### PR TITLE
Security Group Collector to AWS SDK v2

### PR DESCRIPTION
**Pull request by Nirvikalpa108**
### What does this change?
This continues the work started in PR #95 to move prism to AWS SDK v2

**This is ready to be merged to master now**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -43,6 +43,7 @@ lazy val root = (project in file("."))
       "software.amazon.awssdk" % "lambda" % awsVersionTwo,
       "software.amazon.awssdk" % "auth" % awsVersionTwo,
       "software.amazon.awssdk" % "sts" % awsVersionTwo,
+      "software.amazon.awssdk" % "ec2" % awsVersionTwo,
```
Comment (sihil) [inline build.sbt:46]: Note that this is likely to cause a conflict with other EC2 based collectors so you'll likely have to carry out some rebases and deal with the conflicts.

**Review by sihil — APPROVED** [review]
LGTM

**Comment by Nirvikalpa108** [comment]
Testing diff:
`$ diff -I href <(jq -S '.data | .["security-groups"] | sort_by(.arn)' pSec.json) <(jq -S '.data | .["security-groups"] | sort_by(.arn)' cSec.json)`

```
<           "{CidrIp: xxx.xxx.xxx.xxx/xx,}",
>           "IpRange(CidrIp=xxx.xxx.xxx.xxx/xx)",
```

File: app/collectors/securityGroup.scala [hunk 2]
```diff
@@ -21,46 +19,47 @@ class SecurityGroupCollectorSet(accounts: Accounts, prismController: Prism) exte
...
-        rule.getIpProtocol.replace("-1","all"),
-        Option(rule.getFromPort).map(_.toInt),
-        Option(rule.getToPort).map(_.toInt),
-        rule.getIpv4Ranges.asScala.toSeq.map(_.toString).sorted.wrap,
-        rule.getIpv6Ranges.asScala.toSeq.map(_.toString).sorted.wrap,
+        rule.ipProtocol.replace("-1","all"),
+        Option(rule.fromPort).map(_.toInt),
+        Option(rule.toPort).map(_.toInt),
+        rule.ipRanges.asScala.toSeq.map(_.toString).sorted.wrap,
+        rule.ipv6Ranges.asScala.toSeq.map(_.toString).sorted.wrap,
```
Comment (sihil) [inline app/collectors/securityGroup.scala:35]: I think that this is the best way of solving this - the IP range object also includes a description which we previously didn't deal with.
```suggestion
        rule.ipRanges.asScala.toSeq.map(_.cidrIp).sorted.wrap,
        rule.ipv6Ranges.asScala.toSeq.map(_.cidrIpv6).sorted.wrap,
```

Comment (Nirvikalpa108) [reply app/collectors/securityGroup.scala]: This is looking a lot better now:
```
<           "{CidrIp: xx.xx.xxx.x/xx,}"
---
>           "xx.xx.xxx.x/xx"
```

**Pull request merged**